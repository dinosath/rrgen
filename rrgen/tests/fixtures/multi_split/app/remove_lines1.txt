this is a mod file
Delete this line
foo
Delete this line
bar