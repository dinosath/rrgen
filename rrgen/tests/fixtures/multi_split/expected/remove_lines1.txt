this is a mod file
foo
bar